signalfx/pyformance/metadata.py

```diff
@@ -0,0 +1,41 @@
+class MetricMetadata(object):
```

Comment:
All you files need copyright statements at the top

Reply:
I've added copy right statements to all of the new files